webapp/components/Editor/Editor.vue

```diff
@@ -279,6 +320,83 @@ export default {
               return fuse.search(query)
             },
           }),
+          new Hashtag({
+            // a list of all suggested items
+            items: () => {
+              return this.hashtags
+            },
+            // is called when a suggestion starts
+            onEnter: ({ items, query, range, command, virtualNode }) => {
```

Comment:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-07-01T22:44:43Z" title="Tuesday, July 2nd 2019, 12:44:43 am +02:00">Jul 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -279,6 +300,82 @@ export default {
               return fuse.search(query)
             },
           }),
+          new Hashtag({
+            // a list of all suggested items
+            items: () => {
+              return this.hashtags
+            },
+            // is called when a suggestion starts
+            onEnter: ({ items, query, range, command, virtualNode }) => {
```


Here seems to be alot of duplicate code - is it possible to make an mention/tag-agnostic component?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-02T06:08:05Z" title="Tuesday, July 2nd 2019, 8:08:05 am +02:00">Jul 2, 2019</time>_

---


I have an own component  in mind for the future. @ulfgebhardt 
With @appinteractive help I tried to make a class out of mention/hasgtag, but I could not get it to work, because the functions here are reactive and deep connected to the editor component. My skills in JavaScript are not good enough, may be.

I played around with this to long now and like to try that later.

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-07-02T14:43:42Z" title="Tuesday, July 2nd 2019, 4:43:42 pm +02:00">Jul 2, 2019</time>_

---


Ok - awesome!